pkg/daemon/update.go

```diff
@@ -486,6 +483,14 @@ func (dn *Daemon) update(oldConfig, newConfig *mcfgv1.MachineConfig) (retErr err
 			}
 		}
 	}
+	return nil
+}
```

Comment:
similar comment to here - i dont really understand why this is necessary or how it fits into things?